lib/vehicle.rb

```diff
@@ -22,4 +26,19 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def assign_registration_date
+    @registration_date = Date.today
+  end
+
+  def assign_plate_type
+    if antique?
+      @plate_type = :antique
+    elsif
+      electric_vehicle?
+      @plate_type = :ev
+    else
+      @plate_type = :regular
+    end
+  end
```

Comment:
I really like that you added these helper methods to this class. Make sure when you do that though, that you are testing these methods in the vehicle_spec file. You are testing them within register_vehicle, that's an integration test. We also want to test them in isolation as a unit test.